lang/en.json

```diff
@@ -6,6 +6,8 @@
 
         "SETTINGS.HypeTrackEnableN": "Enable Hype Track",
         "SETTINGS.HypeTrackEnableH": "Enable the ability to set a Track for an Actor",
+        "SETTINGS.restartHypeTracksN": "Restart Hype Track",
```

Comment:
Please captialise the first letter of each translation key-part.
This should be `"SETTINGS.RestartHypeTracksN"`